ui-admin/src/study/participants/enrolleeView/EnrolleeProfile.tsx

```diff
@@ -31,6 +32,9 @@ export default function EnrolleeProfile({ enrollee, studyEnvContext, onUpdate }:
   const [isDirty, setIsDirty] = useState<boolean>(false)
   const [editedProfile, setEditedProfile] = useState<Profile>(cloneDeep(enrollee.profile))
   const [showJustifyAndSaveModal, setShowJustifyAndSaveModal] = useState<boolean>(false)
+  const portalEnv = studyEnvContext.portal.portalEnvironments.find((env: PortalEnvironment) =>
+    env.environmentName === studyEnvContext.currentEnv.environmentName)
+  const supportedLanguages = portalEnv?.supportedLanguages || []
```

Comment:
should supported languages be more easily grabbed with a hook? It seems like something we would need to often have these conditional checks on

Reply:
That's a good idea- I'll clean this up as part of https://broadworkbench.atlassian.net/browse/JN-863 since it's related